pkg/security/tests/process_test.go

```diff
@@ -1627,10 +1627,10 @@ EOF
 echo "Back to bash"`, python, python),
 		},
 		{
-			name: "regular exec with interpreter rule",
+			name: "regular exec without interpreter rule",
 			rule: &rules.RuleDefinition{
-				ID:         "test_regular_exec_with_interpreter_rule",
-				Expression: fmt.Sprintf(`exec.file.name == "perl" && exec.interpreter.file.name == "perl"`),
+				ID:         "test_regular_exec_without_interpreter_rule",
```

Comment:
The intent of this test was to make sure that perl is in both the exec and the interpreter, since perl is present in both. What's confusing is that both the original and the new rule both work?

Reply:
ohh i'm seeing the changes in 8f90b6e3c3ef6eb8e0c97d7a275415d40237eb63 now